.github/workflows/linux.yml

```diff
@@ -66,6 +66,7 @@ jobs:
                 -D CMAKE_BUILD_TYPE=${{ env.BUILD_TYPE }} \
                 -D WITH_VCPKG=ON \
                 -D WITH_SPIX=OFF \
+                -D WITH_NFC=OFF \
```

Comment:
Is there a reason for this?

Reply:
@m-kuhn , yeah, we need to toggle this off for linux and macos (and to make it consistent windows) as that requires a pcsclite library which isn't available, leading to missing .so linking error.

Reply:
hmmm... prepackaged qt libs ...

Reply:
I've looked but didn't find the lib